specification/documentation/callbacks/message-subscription-openapi-spec.json

```diff
@@ -0,0 +1,230 @@
+{
+  "openapi": "3.1.0",
+  "info": {
+    "version": "1.0",
+    "title": "Callback Example",
+    "description": "This is an example OpenAPI specification detailing an endpoint to be created by a partner who has integrated with the Communications Manager API and wishes to use the callbacks feature. For context, please see the [Communications Manager API](https://digital.nhs.uk/developer/api-catalogue/communications-manager)."
```

Comment:
NHS Notify*

You may notice that the URL on this line redirects to here: https://digital.nhs.uk/developer/api-catalogue/nhs-notify

Reply:
Fixed